docs/parts-kit.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: default
+title: Parts Kit
+nav_order: 3
```

Comment:
do you have to go manual with this nav order?

Reply:
The default is alpha, so this gives the ability to manually control it.